### ISSUE TITLE
Extract an env var to configure client idle timeout to the dispatcher

### ISSUE THREAD

**Issue by pierDipi** [knative-sandbox/eventing-kafka-broker#141]
**Problem**

Extract an env var to configure client idle timeout to the dispatcher.
See: https://github.com/knative-sandbox/eventing-kafka-broker/pull/137#discussion_r481938539

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?
System Integrator

**Exit Criteria**
A measurable (binary) test that would indicate that the problem has been resolved.

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve?
1

**Additional context (optional)**

- https://github.com/knative-sandbox/eventing-kafka-broker/pull/137#pullrequestreview-480651557

/area data-plane
/good-first-issue

**Comment by slinkydeveloper** [comment]
I think we should also allow configuring other flags of the web client, in particular the pooling etc...

**Comment by pierDipi** [comment]
Can you open another issue with all configuration you would like to have?
We probably need a CM more than env vars ...

**Comment by slinkydeveloper** [comment]
> We probably need a CM more than env vars ...

Agree, there are tons of things you might customize for the web client... How about tackling these in this issue? I might have an idea :smile:

**Comment by pierDipi** [comment]
Go for it! :+1:

**Comment by pierDipi** [comment]
We need a similar solution for the HTTP server as well.
